graspologic/cluster/autogmm.py

```diff
@@ -117,6 +119,13 @@ class AutoGMMCluster(BaseCluster):
         If provided, min_components and ``max_components`` must match the number of
         unique labels given here.
 
+    n_init : int, optional (default = None)
```

Comment:
what is the meaning of `None` as opposed to `1`?

Reply:
Sorry @bdpedigo for the late response! `None` does not append any new k-means runs, essentially keeping the autogmm output untouched. If an integer is provided then additional runs are added.

Reply:
i guess all i meant was how is None different from 0? and if they are the same, should just use 0. 

no need to change the name.

Reply:
or rename to `kmeans_n_init` @tliu68 what do you think?

Reply:
I like that. I think that's more informative/clearer.